### PR TITLE
Check-in user guide for w4a16 LLM deployment

### PR DESCRIPTION
**Pull request by lvhan028**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Present how to deploy w4a16 LLM model

## Modification

w4a16.md in both English and Chinese

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/zh_cn/w4a16.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+```shell
+pip install lmdeploy
+```
+
+## 4bit 权重模型推理
+
+你可以直接从 LMDeploy 的 [model zoo](https://huggingface.co/lmdeploy) 下载已经量化好的 4bit 权重模型，直接使用下面的命令推理。也可以根据["4bit 权重量化"](#4bit-权重量化)章节的内容，把 16bit 权重量化为 4bit 权重，然后再按下述说明推理
+
+以 4bit 的 Llama-2-7B 模型为例，可以从 model zoo 直接下载：
```
Comment (pppppM) [inline docs/zh_cn/w4a16.md:15]:
```suggestion
以 4bit 的 Llama-2-chat-7B 模型为例，可以从 model zoo 直接下载：
```

File: docs/zh_cn/w4a16.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+### 第二步：量化权重模型
+
+LMDeploy 使用 AWQ 算法对模型权重进行量化。在执行下面的命令时，需要把步骤1的`$WORK_DIR`传入。量化结束后，权重文件也会存放在这个目录中。然后就可以根据 ["4bit权重模型推理"](#4bit-权重模型推理)章节的说明，进行模型推理。
+
+```shell
+python3 -m lmdeploy.lite.apis.auto_awq \
+  --w_bits 4 \                       # 权重量化的 bit 数
+  --w_group_size 128 \               # 权重量化分组统计尺寸
+  --work_dir $WORK_DIR \             # 步骤 1 保存量化参数的目录
```
Comment (pppppM) [inline docs/zh_cn/w4a16.md:88]:
```suggestion
python3 -m lmdeploy.lite.apis.auto_awq \
  --model $HF_MODEL \
  --w_bits 4 \                       # 权重量化的 bit 数
  --w_group_size 128 \               # 权重量化分组统计尺寸
  --work_dir $WORK_DIR \             # 步骤 1 保存量化参数的目录
```

File: docs/en/w4a16.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+pip install lmdeploy
+```
+
+## 4-bit LLM model Inference
+
+You can download the pre-quantized 4-bit weight models from LMDeploy's [model zoo](https://huggingface.co/lmdeploy) and conduct inference using the following command.
+
+Alternatively, you can quantize 16-bit weights to 4-bit weights following the ["4-bit Weight Quantization"](#4-bit-weight-quantization) section, and then perform inference as per the below instructions.
+
+Take the 4-bit Llama-2-7B model from the model zoo as an example:
```
Comment (pppppM) [inline docs/en/w4a16.md:17]:
```suggestion
Take the 4-bit Llama-2-chat-7B model from the model zoo as an example:
```

File: docs/en/w4a16.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+### Step2: Quantize Weights
+
+LMDeploy employs AWQ algorithm for model weight quantization.
+
+```shell
+python3 -m lmdeploy.lite.apis.auto_awq \
+  --w_bits 4 \                       # Bit number for weight quantization
+  --w_sym False \                    # Whether to use symmetric quantization for weights
+  --w_group_size 128 \               # Group size for weight quantization statistics
+  --work_dir $WORK_DIR \             # Directory saving quantization parameters from Step 1
```
Comment (pppppM) [inline docs/en/w4a16.md:91]:
```suggestion
python3 -m lmdeploy.lite.apis.auto_awq \
  --model $HF_MODEL \
  --w_bits 4 \                       # Bit number for weight quantization
  --w_group_size 128 \               # Group size for weight quantization statistics
  --work_dir $WORK_DIR \             # Directory saving quantization parameters from Step 1
```

File: docs/zh_cn/w4a16.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
+# W4A16 LLM 模型部署
+
+LMDeploy 支持 4bit 权重模型的推理，**对 NVIDIA 显卡的最低要求是 sm80**。
```
Comment (lzhangzz) [inline docs/zh_cn/w4a16.md:3]: 可以详细点: 
```suggestion
LMDeploy 支持 4bit 权重模型的推理，**对 NVIDIA 显卡的最低要求是 sm80** (A100, GTX 30/40系列等)。
```

File: docs/en/w4a16.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+If you wish to interact with the model via web ui, please initiate the gradio server as indicated below:
+
+```shell
+python3 -m lmdeploy.serve.turbomind ./workspace --server_name {ip_addr} ----server_port {port}
+```
+
+Subsequently, you can open the website `http://{ip_addr}:{port}` in your browser and interact with the model
+
+## Inference Performance
```
Comment (pppppM) [inline docs/en/w4a16.md:49]: It's better to show min cpu&gpu memory

Comment (lvhan028) [reply docs/en/w4a16.md:49]: memory cost is added

**Review by lzhangzz — APPROVED** [review]

**Review by pppppM — APPROVED** [review]
LGTM

**Pull request merged**